usaspending_api/api_docs/api_documentation/advanced_award_search/spending_by_award.md

```diff
@@ -0,0 +1,183 @@
+## [Spending by Award](#spending-by-award)
+**Route:** `/api/v2/search/spending_by_award/`
+
+**Method:** `POST`
+
+This route takes award filters and fields, and returns the fields of the filtered awards.
+
+### Request
+fields: Defines what award variables are returned in an array using the Fields provided below.
+
+filters: Defines how the awards are filtered.  The filter object is defined here.  Unique filters are compounded by AND operators.
```

Comment:
Key-value pairs in the filter object are treated as AND operations with each other.

Reply:
its ORs between the same filters, and AND between distinct filters

Reply:
```
"agencies": [
            {
                 "type": "awarding",
                 "tier": "toptier",
                 "name": "Social Security Administration"
            },
            {
                 "type": "awarding",
                 "tier": "subtier",
                 "name": "Social Security Administration"
            },
            {
                 "type": "funding",
                 "tier": "toptier",
                 "name": "Social Security Administration"
            },
            {
                 "type": "funding",
                 "tier": "subtier",
                 "name": "Social Security Administration"
            }
       ],
       "legal_entities": [779928],
```


(agency 1 Or Agency 2 OR agency 3) AND Legal Entity 1

Reply:
updated the comment. pls take another look